content/docs/user-guide/packaging/project-export/project-export-ios.md

```diff
@@ -0,0 +1,101 @@
+
+The export script has 2 primary sections: the function [`export_ios_xcode_project`](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L16) and the [startup code that only runs if the script is invoked by the CLI](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L73). In depth discussion on these two sections will be made in the [Developer Guide](https://docs.o3de.org/docs/engine-dev/) in the future.
+
+
+### Usage
+To use the export script, you can issue the arguments for this script at the same time that you are running the `export-project` command, so long as you are using this script as your designated export script. The arguments specific to the script will be deferred until the script begins running.
+
+The arguments are as follows:
+| Argument Name | Description | Required? |
+| - | - | - |
+| [`--script-help`](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L89) | Display the help information specifically for the export script. | no |
+| [`--build-tools`](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L90) | Specifies whether to build O3DE toolchain executables. This will build AssetBundlerBatch, AssetProcessorBatch. If asset tools have not been built, then assets cannot be processed. | no |
+| [`--tools-build-path`](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L92) | Designates where the build files for the O3DE toolchain are generated. If not specified, default is `<o3de_project_path>/build/tools`. | no |
+| [`--ios-build-path`](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L94) | Designates where the Xcode project file for the project is generated. If not specified, default is `<o3de_project_path>/build/game_ios`. | no |
+| [`--skip-asset-processing`](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L96) | For iOS building and deployment, the export script assumes that assets should be processed and built. If this behavior is not desired, use this flag to disable it. | no |
+| [`--quiet`](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L98) | Suppresses logging information unless an error occurs. | no |
+
+Here is an example usage of this script:
+```
+# On Mac
+$(O3DE_ENGINE_PATH)/scripts/o3de.sh export-project \
+    --export-script $(O3DE_ENGINE_PATH)/scripts/o3de/ExportScripts/export_source_ios_xcode.py \
+    --project-path $(O3DE_PROJECT_PATH) \
+    --ios-build-path $(IOS_OUTPUT_PATH)
```

Comment:
Don't backslash, this causes problems when people try to click on the `copy` button and paste directly. In addition to `O3DE_ENGINE_PATH` and `PROJECT_PATH`, you should also set an environment variable of `$IOS_OUTPUT_PATH` and describe what it is.

```suggestion
export IOS_OUTPUT_PATH=/path/to/ios/build/path

$O3DE_ENGINE_PATH/scripts/o3de.sh export-project --export-script $(O3DE_ENGINE_PATH)/scripts/o3de/ExportScripts/export_source_ios_xcode.py --project-path $PROJECT_PATH --ios-build-path $IOS_OUTPUT_PATH
```